CHANGES.txt

```diff
@@ -1,5 +1,70 @@
 ﻿NOTE: A combined description of changes since NUnit 2.6.3 is under development along with a summary statement of major features. Both will be available in the first Beta release.
 
+NUnit 3.0.0 Beta 1 - In Process
```

Comment:
Yes, will be. I wasn't sure how long reviewing might take. I'd like you to have an opportunity to actually run the code.